doc/crypt.tex

```diff
@@ -6579,6 +6590,9 @@ \subsection{URL--safe 'base64url' encoding}
 
 int base64url_strict_decode(   const char *in,  unsigned long len,
                             unsigned char *out, unsigned long *outlen);
+
+int base64url_sane_decode(   const char *in,  unsigned long len,
```

Comment:
should we add the URL variants to the index?

Reply:
done